pricegraph/fuzz/fuzz_targets/pricegraph.rs

```diff
@@ -59,6 +59,11 @@ fuzz_target!(|arguments: Arguments| {
                     return;
                 }
             }
+            if let Some(hops) = hops {
+                if hops > 30 {
+                    return;
+                }
+            }
             pricegraph.transitive_orderbook(market, hops, spread);
```

Comment:
Could you just
```suggestion
            pricegraph.transitive_orderbook(market, hops % 30, spread);
```

Not sure what works better with the fuzzer...

Reply:
Dunno which works better but yeah let's use that suggestion.

Reply:
Sorry @e00E, I removed the `merge when green` label as I think there is an issue. It looks like the currently:
- We are using the `if` statement and the modulo expression
- We are computing a modulo on an `Option<usize>`, my suggestion was more pseudo-code than real code :see_no_evil:

Reply:
Oops, thanks. Fixed it.